client/blocks/taxonomy-manager/style.scss

```diff
@@ -26,6 +26,10 @@
 	padding: 16px 24px;
 }
 
+.taxonomy-manager .ReactVirtualized__List {
```

Comment:
Tested and works as expected, but my view is that it's preferable to use an internal css hook rather than rely on an external lib's class name*, which could change if ever we update the package.

So, we might add something like this:

```
.taxonomy-manager .virtual-list__grid {
	direction: ltr !important;
}
```
Then in `client/components/virtual-list/index.jsx`, you could just extend the class name

```
		const listClasses = classNames( 'virtual-list__grid', className );
		return (
			<...
						<List
                             ...
							className={ listClasses }
```
*https://www.npmjs.com/package/react-virtualized

Reply:
If I understand this correctly, this is specifically required as an override of `. ReactVirtualized__List`. It probably wouldn't be necessary if React Virtualized properly handled RTL in the first place. 🙂 

See for example this other override:

https://github.com/Automattic/wp-calypso/blob/master/client/blocks/reader-subscription-list-item/style.scss#L16-L19

which links to a [React Virtualized](https://github.com/bvaughn/react-virtualized/issues/454) issue (from 2016).